packages/content/src/relationships/RelationshipCreationRequestContent.ts

```diff
@@ -0,0 +1,27 @@
+import { ISerializable, Serializable, serialize, type, validate } from "@js-soft/ts-serval";
+import { ContentJSON } from "../ContentJSON";
+import { IResponse, Response, ResponseJSON } from "../requests/response/Response";
+
+export interface RelationshipCreationRequestContentJSON extends ContentJSON {
+    "@type": "RelationshipCreationRequestContent";
+    response: ResponseJSON;
+}
+
+export interface IRelationshipCreationRequestContent extends ISerializable {
+    response: IResponse;
+}
+
+@type("RelationshipCreationRequestContent")
+export class RelationshipCreationRequestContent extends Serializable implements IRelationshipCreationRequestContent {
```

Comment:
I think this should be called `RelationshipCreationContent` as there is no request anymore

Reply:
I prefer `RelationshipTemplateRequestsResponses` - that is the only thing it is used for

Reply:
I mean it's used to send the responses to the requests of a template

Reply:
but that's not where the name should come from. The name should indicate how and where it is transferred (see RelationshipTemplateContent)

Reply:
I expect confusion that the transport creationContent is the more generic Serializable

Reply:
but that's EXACTLY how this works. the content defined in content is our best guess what you can send in this content field. Btw - same with RelationshipTemplate.